fiftyone/core/config.py

```diff
@@ -160,6 +160,30 @@ def __init__(self, d=None):
             env_var="FIFTYONE_BULK_WRITE_BATCH_SIZE",
             default=100_000,  # mongodb limit
         )
+        self.default_batcher = self.parse_string(
+            d,
+            "default_batcher",
+            env_var="FIFTYONE_DEFAULT_BATCHER",
+            default="latency",
+        )
+        self.batcher_static_size = self.parse_int(
+            d,
+            "batcher_static_size",
+            env_var="FIFTYONE_BATCHER_STATIC_SIZE",
+            default=100,
+        )
+        self.batcher_target_size = self.parse_int(
```

Comment:
okay one final idea, take it or leave it: you might consider `batcher_target_size_bytes` here just to clarify that "size" means bytes here, not the same as `batcher_static_size` or `default_batch_size`. As is, the different semantics could be confusing.

Reply:
agree. and it _just_ fits in the table too!

Reply:
lol we're never increasing the width of this table again so we're now limited to 26^current_width different config settings 🤣

Reply:
I have some ideas for configuration upgrades that could fix this but it is not this day